lib/js/jscore.nim

```diff
@@ -110,3 +110,15 @@ proc parse*(l: JsonLib, s: cstring): JsRoot {.importcpp.}
 since (1, 5):
   func debugger*() {.importjs: "debugger@".}
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
+
+  template jsexport*(symbol: untyped; alias = "") =
+    ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
+    runnableExamples("-b:js -r:off"):
+      proc example = echo "This is exported as 'default' in JavaScript"
+      let example2 = "This is exported as 'another' in JavaScript"
+      const example3 = "This is exported as 'example3' in JavaScript"
+      jsexport(example, alias = "default")  ## Alias for `export { symbol as default };`
```

Comment:
is "default" special?
if so, then add a comment to explain.

if not then just showing
```nim
jsexport(example2, alias = "someName") ## Alias for `export { symbol as someName };`
```
would be less confusing